course_catalog/constants.py

```diff
@@ -432,3 +432,28 @@ class PrivacyLevel(Enum):
     "Educational Technology": ["Education & Teacher Training", "Computer Science"],
     "Higher Education": ["Education & Teacher Training"],
 }
+
+VALID_TEXT_FILE_TYPES = [
+    "pdf",
+    "htm",
+    "html",
+    "txt",
+    "doc",
+    "docx",
+    "xls",
+    "xlsx",
+    "ppt",
+    "pptx",
+    "xml",
+    "json",
+    "rtf",
+    "ps",
```

Comment:
@Ferdi any file types missing from this list that you'd like to have text extracted from?